app/package.json

```diff
@@ -8,7 +8,8 @@
     "start": "next start -p 8080",
     "lint": "next lint",
     "test": "jest",
-    "test:watch": "jest --watch"
+    "test:watch": "jest --watch",
+    "test-ci": "jest --ci --coverage"
```

Comment:
Can we rename `"test-ci":  "jest --ci --coverage"` to `test:coverage: "jest --coverage"` and make it works? Thanks!

Reply:
Will need to update the .yml file too.

Reply:
it works, done. thank you so much